docs/controls/wpf-winforms/InkCanvas.md

```diff
@@ -12,7 +12,7 @@ The **InkCanvas** control provides a surface for Windows Ink-based user interact
 ![InkCanvas example](../../resources/images/Controls/InkCanvas.png)
 
 > [!NOTE]
-> This control is currently available as a developer preview for Windows 10, version 1903, and later. Although we encourage you to try out this control in your own prototype code now, we do not recommend that you use it in production code at this time. For more information, see the [XAML Islands feature roadmap](/windows/uwp/xaml-platform/xaml-host-controls#feature-roadmap). If you have feedback about this control, create a new issue in the [Microsoft.Toolkit.Win32 repo](/windows-toolkit/Microsoft.Toolkit.Win32/issues) and leave your comments there. If you prefer to submit your feedback privately, you can send it to XamlIslandsFeedback@microsoft.com.
+> If you have feedback about this control, create a new issue in the [Microsoft.Toolkit.Win32 repo](/windows-toolkit/Microsoft.Toolkit.Win32/issues) and leave your comments there.

```

Comment:
@marb2000 since work isn't being done on that repo and the plan is to point folks to WinUI 3, should we point them to the WinUI repo instead for the future like we're starting to do with DataGrid?

Reply:
I do believe it's a great idea to start to redirect the users to the WinUI repo.

Reply:
I updated all these references in the XAML Islands articles to refer customers to [microsoft-ui-xaml repo](https://github.com/microsoft/microsoft-ui-xaml/issues) instead of the Microsoft.Toolkit.Win32 repo.